Pod/Classes/APMAlertAnimation.swift

```diff
@@ -45,22 +45,23 @@ class APMAlertAnimation: NSObject, UIViewControllerAnimatedTransitioning {
                 completion: {
                     finished in
                     UIView.animate(withDuration: 0.2,
-                            animations: {
-                                alertController.alertView.transform = CGAffineTransform.identity
-                            },
-                            completion: {
-                                finished in
-                                if (finished) {
-                                    transitionContext.completeTransition(true)
-                                }
-                            })
+                        animations: {
+                            alertController.alertView.transform = CGAffineTransform.identity
+                        },
+                        completion: {
+                            finished in
+                            if (finished) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/b2d5b91114f6a23c8648778f00f62f7d3e83fecd/Pod/Classes/APMAlertAnimation.swift#L53"></span>if,for,while,do statements shouldn’t wrap their conditionals in parentheses.</td>
    </tr>
  </tbody>
</table>